### DISCUSSION TITLE
Internal Server Error in Organization's Packages list page

### DISCUSSION THREAD

**Discussion by volodymyr-borodin** [community/community#26376] [Actions]
<p>Internal Server Error in Organization’s Packages list page <a href="https://github.com/orgs/ghosts-network/packages" rel="noopener nofollow ugc">https://github.com/orgs/ghosts-network/packages</a></p>

**Comment by clarkbw** [comment]
<p>Hello! I’m not seeing the error, are you still seeing this?</p>

**Comment by volodymyr-borodin** [comment, marked as answer]
<p>Support has helped me. Error had appeared because of organization renaming</p>
